library/core-components/components/typography/typography.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ComponentProps } from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import { Typography } from './typography';
+
+export default {
+  component: Typography,
+  title: `Core Components/Typography`,
+  parameters: {
+    badges: [BADGE.EXPERIMENTAL],
+    componentSubtitle:
+      'This Polymorphic component provides an interface to apply styles and semantics to text content.',
+  },
+  argTypes: {
+    children: {
+      type: { name: 'string', required: true },
+      defaultValue: 'Hello World!',
+      control: {
+        type: 'text',
+      },
+    },
+    as: {
+      defaultValue: 'p',
+    },
+    align: {
+      defaultValue: 'left',
+    },
+    color: {
+      defaultValue: 'var(--color-text-on-base-primary)',
+    },
+    font: {
+      defaultValue: 'var(--typography-body-md)',
+    },
+  },
+} as ComponentMeta<typeof Typography>;
+
+const Template: ComponentStory<typeof Typography> = (
+  args: ComponentProps<typeof Typography>
+) => <Typography {...args} />;
+
+export const Default: typeof Template = Template.bind({});
+Default.args = {};
+
+export const Example: ComponentStory<typeof Typography> = () => (
+  <div>
+    <Typography font="var(--typography-heading-xl)">Title</Typography>
```

Comment:
I like how flexible the Typography component is to use with color and font props. Would be cool to see a variant prop in the future that maps font and color.

Reply:
I don't think so - that's why it's a random thought for the future; it's good to have what you built - simple and flexible.